backend/src/packages/chaiNNer_standard/image/io/save_image.py

```diff
@@ -66,7 +66,7 @@ class JpegSubsampling(Enum):
     icon="MdSave",
     inputs=[
         ImageInput(),
-        DirectoryInput(has_handle=True),
+        DirectoryInput(must_exist=False, has_handle=True),
```

Comment:
Why not change that here as well?

Reply:
What are you talking about?

Reply:
My bad, I misread this. Don't do code reviews at 4am 😂